docs/0install/setup.md

```diff
@@ -0,0 +1,28 @@
+The 0install distribution of the client is to allow zanata-cli to be available on multiple platform. This distribution requires [0install](http://0install.net/) to run.
+
+## Prerequisites
+
+1. Install [0install](http://0install.net/injector.html) on your machine.
```

Comment:
This should probably be clarified for Mac, because the OSX page is dominated by "Source Installation".  I think @lukebrooker had to use the pre-compiled package bundle, which is easy to miss amongst all the information about building from source.